powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java

```diff
@@ -205,7 +205,7 @@ private void processDocuments(final TestContext context) throws Exception {
         }
 
         assertIdentityVerificationStateWithRetries(
-                new IdentityVerificationState(IdentityVerificationPhase.DOCUMENT_VERIFICATION, IdentityVerificationStatus.IN_PROGRESS));
+                new IdentityVerificationState(IdentityVerificationPhase.DOCUMENT_UPLOAD, IdentityVerificationStatus.IN_PROGRESS));
```

Comment:
@kober32 Here you can see that after upload of the first document, you should expect `DOCUMENT_UPLOAD / IN_PROGRESS`